src/api.js

```diff
@@ -0,0 +1,23 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+export const getTasks = () => {
+  return fetch(`${BASE_URL}/todos`)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      return response.json();
+    });
```

Comment:
Alternatively, you can make a universal function to fetch data, in order not to repeat the same code block